lib/please_cli/please_cli/config.py

```diff
@@ -616,6 +616,64 @@
             },
         ]
     },
+    'staticanalysis/frontend': {
+        'run': 'NEUTRINO',
+        'run_options': {
+            'port': 8010,
+            'envs': {
+                'CONFIG': 'dev',
+            },
+        },
+        'requires': [],
+        'checks': [
+            ('Checking code quality', 'yarn lint'),
+            ('Running tests', 'yarn test'),
+        ],
+        'deploys': [
+            {
+                'target': 'S3',
+                'options': {
+                    'testing': {
+                        's3_bucket': 'static-analysis-testing-frontend',
+                        'url': 'https://static-analysis.testing.moz.tools',
```

Comment:
@garbas I'd like your review on the deploy configuration, especially for the s3_bucket (who creates them ?) and the cloudfront DNS

Reply:
@La0 we are in the process of migrating this responsibility to releng team. so we would be in charge (at least for our stuff) of creating cloudfront DNS. We are currently waiting that everybody returns from PTO. Right now the process is manual, but we will automate (and document it).

Add relevant entries to https://github.com/garbas/build-cloud-tools/blob/master/terraform/base/route53.tf and ping me in the PR. I will merge and deploy it for you.

Reply:
above is for domain stuff.

For S3 bucket + cloudfront we create this manually for now. I hope that once we take over terraform in above, we would also automate this. Do you want me to create this buckets for you?